src/main/java/spring_framework/App01Gaming.java

```diff
@@ -0,0 +1,18 @@
+package spring_framework;
```

Comment:
Use proper package name like `com.marianbastiurea.learnspringframework`.
Organize the project in chapters like you did in `java-fun` project

Reply:
I tried when I started this project but start.spring.io didn't want to let me to do that

Reply:
create new packages

Reply:
done